de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/AbstractIncomingProjectNegotiation.java

```diff
@@ -219,7 +222,8 @@ protected void cleanup(IProgressMonitor monitor, Map<String, IProject> projectMa
      * functionality. This will enable a specific Queuing mechanism per
      * TransferType (see github issue #137).
      */
-    for (IProject project : projectMapping.values()) session.disableQueuing(project);
+    for (IReferencePoint referencePoint : referencePointMapping.values())
```

Comment:
I think it would be less verbose if you just shortcut to `refPoint`, it would still be very clear...

minor thing, dont worry about it